renv.lock

```diff
@@ -760,14 +765,14 @@
     },
     "glue": {
       "Package": "glue",
-      "Version": "1.6.2",
+      "Version": "1.7.0",
```

Comment:
A few R dependencies got updated as part of the lightsnip update (https://github.com/ccao-data/lightsnip/pull/10), which ended up suggesting an upgrade to the latest lightgbm release (4.3.0). I went ahead and allowed the upgrade and it doesn't seem like any breaking changes were introduced that affect us.